ws/src/can_controller/src/velocity_to_can.cpp

```diff
@@ -0,0 +1,97 @@
+#include <cstdio>
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <tuple>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+#include "geometry_msgs/Twist.h"
+
+// TODO: Install the following libraries:
+#include "ctre/Phoenix.h"
+#include "ctre/phoenix/platform/Platform.h"
+#include "ctre/phoenix/unmanaged/Unmanaged.h"
+#include "ctre/phoenix/cci/Unmanaged_CCI.h"
+
+using namespace std::chrono_literals;
+using namespace ctre::phoenix;
+using namespace ctre::phoenix::platform;
+using namespace ctre::phoenix::motorcontrol;
+using namespace ctre::phoenix::motorcontrol::can;
+
+// TODO: Update these port numbers!
+TalonSRX talLeft(1);
+TalonSRX talRght(0);
+
+talLeft.ConfigSelectedFeedbackSensor(...); // TODO: finish filling out, though it may be unnecessary.
+
+int kPIDLoopIdx = 0;
+int kTimeoutMs = 30;
+
+talLeft.Config_kF(kPIDLoopIdx, 0, kTimeoutMs); // TODO: Figure out PIDF values.
+talLeft.Config_kP(kPIDLoopIdx, 0, kTimeoutMs); // TODO: this might have to be moved to main, alongside the ConfigSelectedFeedbackSensor()
+talLeft.Config_kI(kPIDLoopIdx, 0, kTimeoutMs);
+talLeft.Config_kD(kPIDLoopIdx, 0, kTimeoutMs);
+
+// TODO: we will need to tune the velocity PIDF values according to https://phoenix-documentation.readthedocs.io/en/latest/ch16_ClosedLoop.html, probably.
+
+void initDrive()
+{
+	talRght.SetInverted(false); /* both talons should blink green when driving forward */
+}
+
+void drive(double left, double right)
+{
+	talLeft.Set(ControlMode::Velocity, left); // TODO: Update controlmode to be accurate
+	talRght.Set(ControlMode::Velocity, right);
+}
+
+class TalonNode : public rclcpp::Node
+{
+  public:
+    TalonNode()
+    : Node("talon_interface") // TODO: Replace node name
+    {
+      subscription_ = this->create_subscription<tuple>("PLACEHOLDER_VELOCITY_TOPIC", 10, std::bind(&PathfinderSubscriber::topic_callback, this, _1));
```

Comment:
Shouldn't this be `cmd_vel`

Reply:
Also why is it a tuple?

Reply:
Tuple is pretty arbitrary, but seems to work fine, and the velocity topic really just depends on what we end up naming it...

Reply:
ROS standard is to use `cmd_vel` as your velocity topic and `geometry_msgs/Twist` as your message type, we should follow standard as that's what sim and all the other parts of this project are expecting